### PR TITLE
Merge developer/swann to /develop

### PR DESCRIPTION
**Pull request by swannobi**
Fixes for Garpr and image search. New meme. Workaround to allow privileged users to voteskip normally when listening to audio. Initial work on Taiga. Extension to remindme for repeat reminders.

### CODE REVIEW

File: cogs/garpr.py [hunk 1]
```diff
@@ -357,12 +364,12 @@ def check_files(resources_folder):
                 {
                     "region" : "northcarolina",
                     "tournaments on record" : 0,
-                    "rank emotes" : [":pineapple:", ":apple:", ":tangerine:", ":cherries:"],
-                    "rank colors" : [discord.Colour.dark_green().value,discord.Colour.gold().value,discord.Colour.light_grey().value,discord.Colour.purple().value,10061926]
+                    "rank emotes" : [":pineapple:", ":apple:", ":tangerine:", ":cherries:", ""],
+                    "rank colors" : [discord.Colour.gold().value,discord.Colour.dark_green().value,discord.Colour.light_grey().value,discord.Colour.purple().value,10061926]
```
Comment (Savestate2A03) [inline cogs/garpr.py]: I think you should split these up into multiple lines for readability
```
"rank colors" : [
    value,
    value,
    value,
    value
]
```

File: cogs/memes.py [hunk 2]
```diff
@@ -25,6 +25,11 @@ def __init__(self, bot):
         with open('data/memes/lex.png', 'rb') as f:
             await self.bot.send_file(ctx.message.channel, f)
 
+    @commands.command(pass_context=True)
+    async def collin(self, ctx):
+        """F this"""
+        await self.bot.say("https://gfycat.com/AffectionateGenerousBarebirdbat")
+
     @commands.group(name="smith", pass_context=True)
```
Comment (Savestate2A03) [inline cogs/memes.py]: is `name="smith"` required here? I don't see any of the other meme commands following this format

Comment (swannobi) [reply cogs/memes.py]: Good catch; old code. I'll fix that.

File: cogs/garpr.py [hunk 3]
```diff
@@ -171,6 +171,7 @@ def _get_playerid(self, player : str):
             "and "+str(stats["losses"])+" losses ("+ratio+")")
         except KeyError as e:
             print(e)
+            await self.bot.add_reaction(message=ctx.message, emoji="❌")
```
Comment (Savestate2A03) [inline cogs/garpr.py:175]: i really like this

**Pull request merged**